server/src/main/java/org/elasticsearch/bootstrap/Elasticsearch.java

```diff
@@ -134,20 +131,6 @@ private static Bootstrap initPhase1() {
         final boolean useEntitlements = true;
```

Comment:
As far as I can tell this is initialized to `true` and then never modified. Can we get rid of this?

Reply:
Yes, I plan to do that in a followup, there are some other things using it.